src/core/src/bootstrap/Constants.py

```diff
@@ -53,6 +53,7 @@ def __iter__(self):
 
     class Paths(EnumBackport):
         SYSTEMD_ROOT = "/etc/systemd/system/"
+        EULA_SETTINGS = "/var/lib/azure/linuxpatchextension/patch.eula.settings"
```

Comment:
Minor, for long term: Good to split this out into SystemPaths and AzGPSPaths

Reply:
done